findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js

```diff
@@ -131,7 +131,7 @@ export default class InjuryPainCare extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
```

Comment:
Card and accordion fonts aren't applied on Android:
![image](https://user-images.githubusercontent.com/22266870/79055462-f763f480-7c12-11ea-9229-7a7e85472426.png)